pipeline/test/test_processor.py

```diff
@@ -0,0 +1,788 @@
+                3: False,
+                4: True,
+                5: False,
+                6: False,
+                7: False,
+                8: False,
+                9: False,
+            },
+            "last_list_ix_bool": {
+                0: False,
+                1: False,
+                2: False,
+                3: False,
+                4: False,
+                5: False,
+                6: False,
+                7: True,
+                8: False,
+                9: True,
+            },
+        }
+    )
+
+    expected_out = [
+        "\n<li1>\n<Lbl>(a)<\\Lbl><LBody>shall develop a standard contract providing for net energy metering, and shall, upon request, make this contract available to eligible customer-generators;<\\LBody>\n",
+        "<Lbl>(b)<\\Lbl><LBody>shall prepare appropriate technical standards for grid connection of renewable energy systems, and inspect and provide a license for those renewable energy installations that meet the technical standards developed by P.P.U.C. and the other provisions of this legislation. Issuance of a license shall be solely to show that the P.P.U.C. has approved the interconnection of the customer=s renewable energy system and the P.P.U.C. grid and shall not be interpreted to impose liability or approval by the P.P.U.C. for any part of the renewable energy system, its design, or its method of implementation. The technical standards imposed will be based solely on those necessary to ensure the safety of P.P.U.C. personnel and for the maintenance of P.P.U.C, power quality. Standards and technical requirements shall be consistent with existing technical practices for similar types of installations in the United States, Australia, or the European Union.<\\LBody>\n",
+        "\n<li2>\n<Lbl>(1)<\\Lbl><LBody>A licensee shall inform the P.P.U.C. of any proposed technical changes to the renewable energy system that affects either the maximum power output or the components that provide the interconnection between the renewable energy system and the P.P.U.C. grid and will, under the licensing agreement, not make those changes without P.P.U.C. approval.<\\LBody>\n",
+        "<Lbl>(2)<\\Lbl><LBody>The failure of a licensee to promptly inform the P.P.U.C. in writing of any technical changes to the renewable energy system that affects any of the above may, at the P.P.U.C. discretion, result in a fine of not more than two hundred dollars ($200).<\\LBody>\n\n<\\li2>\n",
+        "<Lbl>(c)<\\Lbl><LBody>shall, at its own-expense, make available to each of its eligible customer generators who have installed a net metering system the meter (or set of meters) that is needed to determine the net flow of electricity both into and out of the electricity grid;<\\LBody>\n\n<\\li1>\n",
+    ]
+
+    actual_out = adobe_list_postprocessor._format_semantic_lists(in_df)
+    assert actual_out == expected_out
+
+
+def test_adobe_list_processor_pprinter(adobe_list_postprocessor):
```

Comment:
This test is more so you can just see what the pretty printed output would look like (if used by, say, the API) when running the test suite with the arguments -rP